Detectors/TRD/simulation/src/Trap2CRU.cxx

```diff
@@ -724,14 +758,17 @@ void Trap2CRU::convertTrapData(o2::trd::TriggerRecord const& triggerrecord, cons
           while (mDigits[mDigitsIndex[mCurrentDigit]].getMCM() == currentMCM &&
                  mDigits[mDigitsIndex[mCurrentDigit]].getROB() == currentROB &&
                  mDigits[mDigitsIndex[mCurrentDigit]].getDetector() == currentDetector) {
+            LOG(info) << __func__ << " " << __LINE__;
             LOG(debug) << " on index of : " << mDigitsIndex[mCurrentDigit] << " wuf channel=" << mDigits[mDigitsIndex[mCurrentDigit]].getChannel();
             mCurrentDigit++;
             digitcounter++;
             adccounter++;
             if (digitcounter > 22) {
+              LOG(info) << __func__ << " " << __LINE__;
```

Comment:
Actually, why is this 22? I would understand 10, 11, 30 or 31...

Reply:
21 digits per mcm and digitcounter has just been incremented before so 22.

Reply:
Ahh, digits as in ADCs. Which is in line with how we defined o2::trd::Digit, so that's ok.